src/spikeinterface/core/generate.py

```diff
@@ -317,6 +318,91 @@ def generate_snippets(
 
 
 ## spiketrain zone ##
+def synthesize_poisson_spike_vector(
+    num_units=20,
+    sampling_frequency=30000.0,
+    duration=60,
+    refractory_period_ms=4.0,
+    firing_rates=3.0,
+    seed=0,
+):
+    """
+    Generate random spike frames for neuronal units based on a Poisson process.
+
+    This function simulates the spike frames of a number of neuronal units, each firing according to
+    a Poisson process. The spike times are discretized to align with a given sampling frequency.
+
+    Parameters
+    ----------
+    num_units : int, optional
+        Number of neuronal units to simulate (default is 20).
+    sampling_frequency : float, optional
+        Sampling frequency in Hz (default is 30000.0).
+    duration : float, optional
+        Duration of the simulation in seconds (default is 60).
+    refractory_period_ms : float, optional
+        Refractory period between spikes in milliseconds (default is 4.0).
+    firing_rates : float or array_like, optional
+        Firing rate(s) in Hz. Can be a single value or an array of firing rates for each unit
+        (default is 3.0).
```

Comment:
```suggestion
    num_units : int, default: 20
        Number of neuronal units to simulate 
    sampling_frequency : float, default: 30000.0
        Sampling frequency in Hz 
    duration : float, default: 60.0
        Duration of the simulation in seconds
    refractory_period_ms : float, default: 4.0
        Refractory period between spikes in milliseconds
    firing_rates : float or array_like, default: 3.0
        Firing rate(s) in Hz. Can be a single value for all units or an array of firing rates with
        each element being the firing rate for one unit
```
Just enforcing docstring style :)

I made the last one a little more verbose (not that I thought yours was unclear, but I sometimes it is nice to spell it out. You can feel free to revert that part if you want.